src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java

```diff
@@ -103,17 +100,23 @@ private void process() {
 				System.out.println("How many records do you want to insert ? : ");
 				int custCount = scanner.nextInt();
 
-				CustomerDataProcessor custProcessor = new CustomerDataProcessor(scanner, custDao);
 				int custRowCount = 1;
 				for (int i = 0; i < custCount; i++) {
-					custProcessor.process(custRowCount);
+					custProcessor.insertProcess(custRowCount);
 					custRowCount++;
 				}
 			} else if (operationTypeCustomer == 2) {
 				System.out.println("fetching customer records from the database...");
-				custDao.customerSelectData();
+				custDao.selectCustomerData();
```

Comment:
use processor to call selectCustomerData not dao

Reply:
it is not my module to change

Reply:
this is not my module to change